tests/app/forms/validation/test_mutually_exclusive_validator.py

```diff
@@ -57,5 +57,5 @@ def test_mutually_exclusive_mandatory_checkbox(answer_permutations, is_mandatory
     validator(
         answer_values=iter(answer_permutations),
         is_mandatory=is_mandatory,
-        is_only_checkboxes=True,
+        is_only_checkboxes_or_radios=True,
     )
```

Comment:
This test also need to update the parametrize values to include an example that would match a radio. The current one are all a `list` which means it is testing checkbox.

Reply:
Renamed the test since it's not only checkboxes being tested now and added some radio parametrize values to include various radios.